dEdX/include/dEdxAnalyser.h

```diff
@@ -136,15 +171,22 @@ class dEdxAnalyser : public Processor {
   double _cutMomMin=0;
   double _cutMomMax=0;
 
-  bool _plotStuff=true;
+  double _fidMomMin=0;
+  double _fidMomMax=0;
+  double _fidLamMin=0;
+  double _fidLamMax=0;
+  int _fidNHitsMin=0;
+
+  bool _plotStuff=false;
   bool _useLCTPCStyle=false;
   std::string _fileFormat = ".png";
 
   // number of particle species to be separated
   static const int _nPart = 5;
 
-  // pdg numbers of particles, to be specified in ::init() (along with corresponding strings for names and descriptions)
+  // pdg numbers and masses of particles, to be specified in ::init() (along with corresponding strings for names and descriptions)
   std::vector<int> _PDG{};
+  std::vector<double> _Masses{};
```

Comment:
Since the number of particles is fixed and not configurable, why not also fix the pdgs, and masses? Or at least fix the sizes of the arrays (then without `constexpr` for the `std::array`s below).

```cpp
    static constexpr int _nPart{5};
    constexpr std::array<int, _nPart + 1> _PDG = {11, 13, 211, 321, 2212, 10};
    constexpr std::array<double, _nPart> _Masses = {0.000511, 0.10566, 0.13957, 0.49368, 0.93827};
```